packages/cli/test/integration/shared/db/workflows.ts

```diff
@@ -19,12 +19,7 @@ export async function createManyWorkflows(
 	return await Promise.all(workflowRequests);
 }
 
-/**
- * Store a workflow in the DB (without a trigger) and optionally assign it to a user.
- * @param attributes workflow attributes
- * @param user user to assign the workflow to
- */
-export async function createWorkflow(attributes: Partial<WorkflowEntity> = {}, user?: User) {
+export function newWorkflow(attributes: Partial<WorkflowEntity> = {}): WorkflowEntity {
```

Comment:
In `packages/cli/test/integration/import.service.test.ts` I need the workflow without having it in the db yet.

This is in line with the changes made in here: 

https://github.com/n8n-io/n8n/blob/9872849c30ff8fcb2f4d107276c7a664b72e8749/packages/cli/test/integration/shared/db/users.ts#L16-L26